bazel/benchmark/app_bundling/index.bzl

```diff
@@ -86,14 +86,13 @@ def app_bundle(
     esbuild(
         name = name,
         args = _create_esbuild_minify_options(False),
-        **common_esbuild_options
+        **dict(kwargs, **common_esbuild_options)
     )
 
     esbuild(
         name = "%s.debug" % name,
         args = _create_esbuild_minify_options(True),
-        tags = ["manual"],
-        **common_esbuild_options
+        **dict(kwargs, tags = ["manual"], **common_esbuild_options)
```

Comment:
Do we want `tags` to apply here as well rather than always being just `manual`

Reply:
yeah, I wasn't sure about that as well, but just left it like that because I think realistically the `debug` target should not receive any tags. If one adds e.g. `integration-test` to the app bundle target, then I'd expect the Bazel tag filter to only match the actual target and not the debug-one.